apps/user-profile/src/pages/_app.js

```diff
@@ -0,0 +1,83 @@
+import Head from 'next/head';
+import { Router, useRouter } from 'next/router';
+import React from 'react';
+import { ToastContainer } from 'react-toastify';
+import { ThemeProvider } from 'styled-components';
+
+import GlobalStyle from '../styles/globals';
```

Comment:
GlobalStyle should be imported from "@devlaunchers/components/src/styles/styles"